lefthook.yml

```diff
@@ -1,5 +1,14 @@
 pre-commit:
+  parallel: true
   commands:
+    type-check:
+      glob: "src/**/*.{ts,tsx}"
```

Comment:
nit: all tsx files are excluded in tsconfig, right?

Reply:
Ohh yes we actually don't need to check for tsx since we won't be having them in cli 🙌 nice catch, thanks Rinat!